featuretools/tests/entityset_tests/test_es_metadata.py

```diff
@@ -49,37 +51,37 @@ def test_get_forward_entities_deep(es):
     ]
 
 
-def test_get_backward_entities_deep(es):
-    entities = es.get_backward_entities('customers', deep=True)
+def test_get_backward_dataframes_deep(es):
+    dataframes = es.get_backward_dataframes('customers', deep=True)
     path_to_log = backward_path(es, ['customers', 'sessions', 'log'])
     path_to_sessions = backward_path(es, ['customers', 'sessions'])
-    assert list(entities) == [('sessions', path_to_sessions), ('log', path_to_log)]
+    assert list(dataframes) == [('sessions', path_to_sessions), ('log', path_to_log)]
 
 
 def test_get_forward_relationships(es):
     relationships = es.get_forward_relationships('log')
     assert len(relationships) == 2
-    assert relationships[0].parent_dataframe.id == 'sessions'
-    assert relationships[0].child_dataframe.id == 'log'
-    assert relationships[1].parent_dataframe.id == 'products'
-    assert relationships[1].child_dataframe.id == 'log'
+    assert relationships[0].parent_dataframe.ww.name == 'sessions'
+    assert relationships[0].child_dataframe.ww.name == 'log'
+    assert relationships[1].parent_dataframe.ww.name == 'products'
+    assert relationships[1].child_dataframe.ww.name == 'log'
```

Comment:
I think we could grab these names directly from the relationship instead of going through Woodwork?

Reply:
changed all of these checks to look directly at the relationship attribute